### ISSUE TITLE
Accessibility Fixes

### ISSUE THREAD

**Issue by kpwags** [kpwags/new.kpwags.com#68]
The following Accessibility fixes should be put into place

- [x] Add `aria-label` to `nav` elements

> One tip here that I feel a bit embarrassed that I didn’t know is that we should be using the aria-label attribute on nav elements to describe them:

<nav aria-label=“Main menu”>
  
</nav>
